setup.py

```diff
@@ -1,13 +1,12 @@
 from setuptools import setup, find_packages
-import sys, os
 
-version = '1.0'
+version = '1.1'
 
 setup(name='pyzipcode',
       version=version,
       description="query zip codes and location data",
       long_description=open("README.txt").read() + '\n\n' + open('CHANGES.txt').read(),
-      classifiers=[], # Get strings from http://pypi.python.org/pypi?%3Aaction=list_classifiers
+      classifiers=[],  # Get strings from http://pypi.python.org/pypi?%3Aaction=list_classifiers
       keywords='zip code distance',
```

Comment:
This should also have a python_requires parameter to define the supported python versions

Reply:
Oh, good catch! On it.